marshal-configs/sha3-bare-sw.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name" : "sha3-bare-sw",
+  "workdir" : "sha3",
+  "base" : "bare",
+  "host-init" : "build.sh",
+  "bin" : "benchmarks/bare/sha3-sw.riscv",
+  "spike" : "spike-local/bin/spike",
+  "spike-args" : "--extension=sha3",
+  "testing" : {
+    "refDir" : "test-reference/bare"
```

Comment:
Similarly, the reference output must be added to `test-reference/bare` for `marshal test` to succeed.